en/bundle.ftl

```diff
@@ -44,3 +46,13 @@ bundle-feature-one = More than { $num_vpn_servers } servers
 #   $num_vpn_countries (string) - the number of VPN available countries. Examples: 30, 40, 50
 bundle-feature-two = More than { $num_vpn_countries } countries
 bundle-feature-three = Fast and secure network
+
+## VPN and Relay Premium Dashboard Upgrade Banner
+
+bundle-banner-dashboard-header = Introducing: { -brand-name-relay } + { -brand-name-vpn } subscription plan
+# Variables:
+#   $monthly_price (string) - the monthly cost (including currency symbol) for a given plan. Examples: $0.99, 0,99 €
+#   $savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
+bundle-banner-dashboard-body = Upgrade your subscription to get both { -brand-name-relay-premium } Phone Masking + { -brand-name-mozilla-vpn } for { $monthly_price }/month. Upgrade now to get { $savings } off { -brand-name-mozilla-vpn }!
+bundle-banner-dashboard-upgrade-cta = Upgrade now
```

Comment:
For upcoming feature (MPP-2480) 
<img width="372" alt="image" src="https://user-images.githubusercontent.com/13066134/197018298-8da3ef29-bb9c-4d5e-8afa-daaf13df893d.png">